src/sgp/Video.cc

```diff
@@ -99,24 +100,28 @@ void VideoSetFullScreen(const BOOLEAN enable)
 {
 	if (enable)
 	{
-		g_window_flags |= SDL_WINDOW_FULLSCREEN;
+		g_window_flags |= SDL_WINDOW_FULLSCREEN_DESKTOP;
 	}
 	else
 	{
-		g_window_flags &= ~SDL_WINDOW_FULLSCREEN;
+		g_window_flags &= ~SDL_WINDOW_FULLSCREEN_DESKTOP;
```

Comment:
Why the change of type? If the nearest scaling requires the fake fullscreen, and can't work with the mode change, I think it would be better for that to swap out the flags and leave normal fullscreen for the rest. Two things are in my head; is it a performance hit and would it work or nicely degrade on android.

Reply:
The renderer needs to be set to the Desktop resolution to let SDL do the scaling. If we don't set integer scaling, SDL still scales up the image to fullscreen with linear interpolation (like the monitor would otherwise), so I did not see any reason to not use SDL_WINDOW_FULLSCREEN_DESKTOP all the time, so to keep the code simple. It then work similar to the fix in issue #62, when the window is resized, only for fullscreen this time. 
I guess it would not be a performance problem on anything that is not ancient. 

Although as a CRT user you would probably be better served by SDL_WINDOW_FULLSCREEN. So I could change it as proposed.

Reply:
For me it probably makes no difference or is a slight improvement, in case a crash somehow prevents proper video mode restoration. All I could find on the net were reports of multiscreen issues, but for both modes, so meh.

So unless @selaux, @janisozaur or anyone else has any further complaints, this can stay. It would be better in a separate commit though.